imgparse/xmp.py

```diff
@@ -49,6 +49,21 @@ class DJITags(SensorTags):
     YAW = "drone-dji:GimbalYawDegree"
     FOCAL_LEN = "drone-dji:CalibratedFocalLength"
     SELF_DATA = "drone-dji:SelfData"
+    WAVELENGTH_CENTRAL = "Camera:CentralWavelength"
+    WAVELENGTH_FWHM = "Camera:WavelengthFWHM"
+    BANDNAME = "Camera:BandName"
+
+
+class MicaSenseTags(SensorTags):
+    """MicaSense XMP Tags."""
+
+    ROLL = "Camera:Roll"
+    PITCH = "Camera:Pitch"
+    YAW = "Camera:Yaw"
+    FOCAL_LEN = "Camera:PerspectiveFocalLength"
+    WAVELENGTH_CENTRAL = "Camera:CentralWavelength"
+    WAVELENGTH_FWHM = "Camera:WavelengthFWHM"
+    BANDNAME = "Camera:BandName"
```

Comment:
Do you have a sample micasense image that we can add to the tests to make sure these tags work?

Reply:
https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/128f96f0bb5a2d923c00bdf2fdef61f780c3febb Added, removed the roll/pitch/yaw/focal_len for now. I am not using them in anything. Only using Wavelenth and Bandname. I can go back and re-add them if you think, or we can just wait to add them until we actually need them.

Reply:
Confirmed we don't need these for now